labrad/server.py

```diff
@@ -350,6 +350,7 @@ def startup(self, protocol):
             log.err("connection failed, disconnecting")
             traceback.print_exc()
             self.disconnect(e)
+            raise
```

Comment:
@jwenner, if you look a few lines further up, you'll see that this happens inside an `except` clause. A bare `raise` inside `except` just reraises the caught exception: https://docs.python.org/2/reference/simple_stmts.html#raise